website/app/components/doc-landing-callout-blocks.hbs

```diff
@@ -0,0 +1,7 @@
+<ul class="doc-landing-callout-blocks doc-landing-callout-blocks--layout-{{@cols}}cols">
+  {{#each @items as |item|}}
+  <li class="doc-landing-callout-blocks__item">
+    <a class="doc-landing-callout-blocks__link" href={{item.url}}>{{item.text}}</a>
```

Comment:
I don't know that it needs to be addressed in this PR, but manually crafting urls like this feels a bit un-Ember-y to me and there is likely a more idiomatic way to use a normal `{{link-to}}` to accomplish this same thing.

Reply:
these are super-temporary placeholders, they will definitely be replaced when we will build the actual landing pages. but I'll keep in mind it for then. thanks